src/index.ts

```diff
@@ -17,11 +17,27 @@ export const angularModule = angular.module(name, [])
 
         init: function($scope, el): void {
           if (!el.hasAttribute('popgun-listening')) {
-            el.addEventListener('PopgunContentSetup', function(e) {
-              let pop = popgun.getPopFromGroupId((<Element>e.target).getAttribute('popgun-group'));
-              $compile(pop.popOver.element)($scope);
-              $scope.$apply();
-            }, false);
+            el.addEventListener('PopgunContentSetup', (function($compileScope) {
+              return function (e) {
+                  let $newScope = $compileScope.$new();
```

Comment:
Then I suppose that we can document that popgun defaultly creates a child scope, and it can be overwritten by changing a flag.

I'm thinking...:
`AngularPopgunSrvc.int($scope, el, true); // where true represents sharing the same scope passed in`